lib/SwiftDemangle/MangleHack.cpp

```diff
@@ -56,7 +56,7 @@ _swift_mangleSimpleClass(const char *module, const char *class_) {
   size_t moduleLength = strlen(module);
   size_t classLength = strlen(class_);
   char *value = nullptr;
-  if (strcmp(module, swift::STDLIB_NAME) == 0) {
+  if (swift::STDLIB_NAME.compare(module) == 0) {
```

Comment:
These two are just checking for equality, so no need to use `compare` now that these are StringLiterals.

Reply:
I am going to do that in a follow up commit. I am not trying to change anything beyond surface level here.